mozilla-release/mobile/android/base/locales/en-US/cliqz_strings.dtd

```diff
@@ -3,6 +3,9 @@
 <!ENTITY action_ok "OK">
 <!ENTITY action_exit "Exit">
 
+<!-- Onboarding -->
+<!ENTITY firstrun_urlbar_subtext3 "Cliqz - A fast and private browser. Made in Germany">
```

Comment:
Cliqz is hard coded and should not be, this message is totally wrong for Ghostery. @alina-cliqz